plugins/modules/ipa_hostgroup.py

```diff
@@ -192,7 +203,9 @@ def main():
                          description=dict(type='str'),
                          host=dict(type='list', elements='str'),
                          hostgroup=dict(type='list', elements='str'),
-                         state=dict(type='str', default='present', choices=['present', 'absent', 'enabled', 'disabled']))
+                         state=dict(type='str', default='present',
+                         choices=['present', 'absent', 'enabled', 'disabled']),
```

Comment:
Please move `choices` back to the previous line.

Reply:
Moved